index.js

```diff
@@ -5,27 +5,79 @@ const { TimeIt } = require('./percy/util/timing');
 const log = require('./percy/util/log');
 const utils = require('@percy/sdk-utils');
 
+// Collect client and environment information
+const sdkPkg = require('./package.json');
+const CLIENT_INFO = `${sdkPkg.name}/${sdkPkg.version}`;
+
+let clientWdPkg = null;
+try {
+  clientWdPkg = require('wd/package.json');
+} catch { }
+
+try {
+  clientWdPkg = require('webdriverio/package.json');
+} catch { }
+
+let ENV_INFO = `(${clientWdPkg?.name}/${clientWdPkg?.version})`;
+
 async function isPercyEnabled(driver) {
   if (!(await utils.isPercyEnabled())) return false;
 
   return (await driver.getPercyOptions()).enabled;
 }
 
-module.exports = async function percyScreenshot(driver, name, {
-  fullscreen,
-  deviceName,
-  orientation,
-  statusBarHeight,
-  navigationBarHeight,
-  fullPage,
-  screenLengths,
-  ignoreRegionXpaths,
-  ignoreRegionAccessibilityIds,
-  ignoreRegionAppiumElements,
-  customIgnoreRegions,
-  scrollableXpath,
-  scrollableId
-} = {}) {
+async function getElementIdFromElements(type, elements) {
+  if (type === 'wd') return elements.map(e => e.value);
+  /* istanbul ignore next */
+  if (type === 'wdio') return elements.map(e => e.elementId);
+}
+
+async function percyOnAutomate(driver, name, options) {
```

Comment:
can we move this under diff file in percy folder, index.js file should contain percyScreenshot only.